paf-mvp-client-express/src/client-node.ts

```diff
@@ -398,7 +395,7 @@ export class ClientNode extends Node {
         };
         next(error);
       } else {
-        res.statusCode = 204; // OK - no content
+        res.sendStatus(204); // OK - no content
```

Comment:
Setting the statusCode on the response object is not sufficient to terminate the request handling properly. Requests were timing out in integration tests. 
I think the sendStatus method is more suitable here.

Reply:
good catch, thanks! 👍